hubploy/auth.py

```diff
@@ -69,49 +69,65 @@ def registry_auth_gcloud(deployment, project, service_key):
     yield
 
 
-def registry_auth_aws(deployment, project, zone, service_key):
+def registry_auth_aws(deployment, project, zone, service_key = None, ecr_role = None):
```

Comment:
Let's call this parameter 'role' rather than 'ecr_role'? The registry section is already ecr specific.